python/ray/ml/predictors/integrations/tensorflow/tensorflow_predictor.py

```diff
@@ -131,9 +132,9 @@ def build_model(self):
         if isinstance(data, pd.DataFrame):
             if feature_columns:
                 data = data[feature_columns]
-            data = data.values
-
-        tensor = tf.convert_to_tensor(data, dtype=dtype)
+            tensor = convert_pandas_to_tf_tensor(data)
```

Comment:
Hmm I think it will be confusing to the user if the `dtype` that they passed is ignored here. What if we expanded the `convert_pandas_to_tf_tensor()` to accept a `dtype` argument, which will be used instead of the inferred dtype? We're going to want this anyway in a feature PR, when we allow users to explicitly specify the feature tensor and label tensor dtypes in `.to_tf()`.